db/migrate/20191209095548_ensures_unique_accepted_answer_post_id.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class EnsuresUniqueAcceptedAnswerPostId < ActiveRecord::Migration[5.2]
+  def change
+    execute <<~SQL
+      DELETE FROM topic_custom_fields AS tcf1
+      USING topic_custom_fields AS tcf2
+      WHERE tcf1.id > tcf2.id
+         AND tcf1.value = tcf2.value
```

Comment:
this is getting close but not quite there ... I think the condition we want is: 

```
tcf1.id > tcf2.id AND
tcf1.topic_id = tcf2.topic_id AND
tcf1.name = tcf2.name AND
tcf1.name = 'accepted_answer_post_id'
```

We should test the query though by hand, I recommend that on local you have 2 topics, then accept answers on both, then jimmy the database by inserting a row, then run the query to clean up the mess and confirm both topics have a single row vs 2.

Reply:
Yes I thought of it but wasn't sure if one was any better than the other.

This has been tested locally already